app/views/prosecution_cases/show.html.haml

```diff
@@ -0,0 +1,22 @@
+= govuk_page_title(@prosecution_case.prosecution_case_reference)
+
+.govuk-heading-m
+  = t('search.result.defendant.heading')
+%table.govuk-table
+  %caption.govuk-table__caption.govuk-visually-hidden
+    = t('search.result.caption')
+  %thead.govuk-table__head
+    %tr.govuk-table__row
+      %th.govuk-table__header{ scope: 'col' }= t('search.result.defendant.name')
+      %th.govuk-table__header{ scope: 'col' }= t('search.result.defendant.date_of_birth')
+      %th.govuk-table__header{ scope: 'col' }= t('search.result.defendant.maat_number')
```

Comment:
DRY table headers by creating an array within the relevant controller, and looping within the view.

Apply to all tables.

Reply:
At the moment I have avoided too much DRYing out as things may change. Putting presentation logic in the controllers is not great, although there is already some. I will look at sharing partials and or using view helpers. a `presenter`could also solve this problem.